File: loplat SDK Sample for ObjC 2/loplat SDK Sample for ObjC/ViewController.m
//
//  ViewController.m
//  loplat SDK Sample for ObjC
//
//  Created by 상훈 on 01/08/2018.
//  Copyright © 2018 loplat. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self initView];
}

- (void)viewWillAppear:(BOOL)animated {
    [super viewWillAppear:animated];
    
    UIApplication.sharedApplication.statusBarStyle = UIStatusBarStyleDefault;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

- (void)initView {
    self.titleText.font = [UIFont fontWithName:@"AppleSDGothicNeo-Thin" size:20.0];
    self.summaryText.font = [UIFont systemFontOfSize:13.0];
    
    [self.okButton addTarget:self action:@selector(okButtonClicked:) forControlEvents:UIControlEventTouchUpInside];
}

- (void)okButtonClicked:(UIButton*)button {
    [self dismissViewControllerAnimated:YES completion:nil];
}

@end

File: loplat SDK Sample for Swift/loplat SDK Sample for Swift/ViewController.swift
//
//  ViewController.swift
//  loplat SDK Sample for Swift
//
//  Created by 상훈 on 31/07/2018.
//  Copyright © 2018 loplat. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet private weak var titleText: UILabel? = nil
    @IBOutlet private weak var summaryText: UILabel? = nil
    @IBOutlet private weak var okButton: UIButton? = nil

    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.initView()
    }
    
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        self.navigationController?.navigationBar.barStyle = .default
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override var preferredStatusBarStyle: UIStatusBarStyle {
        return .default
    }

    /// 필수 UI 구성요소를 초기화합니다.
    private func initView() {
        self.titleText?.font = UIFont(name: "AppleSDGothicNeo-Thin", size: 20.0)!
        self.summaryText?.font = UIFont.systemFont(ofSize: 13.0)
        
        okButton?.addTarget(self, action: #selector(self.okButtonClicked), for: .touchUpInside)
    }
    
    @objc private func okButtonClicked(_ button: UIButton) {
        self.dismiss(animated: true, completion: nil)
    }
}

